docs/outputs.rst

```diff
@@ -161,40 +198,105 @@ Each row of the file corresponds to one time point found in the
 corresponding :abbr:`BOLD (blood-oxygen level dependent)` time-series
 (stored in ``<output_folder>/fmriprep/sub-<sub_id>/func/sub-<sub_id>_task-<task_id>_run-<run_id>_desc-preproc_bold.nii.gz``).
 
-Columns represent the different confounds: ``csf`` and ``white_matter`` are the average signal
-inside the anatomically-derived :abbr:`CSF (cerebro-spinal fluid)` and :abbr:`WM (white matter)`
-masks across time;
-``global_signal`` corresponds to the mean time series within the brain mask; two columns relate to
-the derivative of RMS variance over voxels (or :abbr:`DVARS (defined in Power, et al. 2012)`), and
-both the original (``dvars``) and standardized (``std_dvars``) are provided;
-``framewise_displacement`` is a quantification of the estimated bulk-head motion;
-``trans_x``, ``trans_y``, ``trans_z``, ``rot_x``, ``rot_y``, ``rot_z`` are the 6 rigid-body
-motion-correction parameters estimated by fMRIPrep;
-if present, ``non_steady_state_outlier_XX`` columns indicate non-steady state volumes with a single
-``1`` value and ``0`` elsewhere (*i.e.*, there is one ``non_steady_state_outlier_XX`` column per
-outlier/volume);
-additional noise components are calculated using :abbr:`CompCor (Component Based Noise Correction)`,
-according to both the anatomical (``a_comp_cor_XX``) and temporal (``t_comp_cor_XX``) variants;
-and the motion-related components identified by
-:abbr:`ICA (independent components analysis)`-:abbr:`AROMA (Automatic Removal Of Motion Artifacts)`
-(if enabled) are indicated with ``aroma_motion_XX``.
+Confound regressors description
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+Basic confouds
+==================
+
+- ``trans_x``, ``trans_y``, ``trans_z``, ``rot_x``, ``rot_y``, ``rot_z`` - the 6 rigid-body motion
+  parameters (3 translations and 3 rotation) estimated relative to a reference image;
+
+- ``csf`` - the average signal within anatomically-derived eroded :abbr:`CSF (cerebro-spinal fluid)` mask;
+
+- ``white_matter`` - the average signal within  the anatomically-derived eroded :abbr:`WM (white matter)` masks;
+
+- ``global_signal`` -  the average signal within the brain mask.
+
+Parameter expansion of basic confounds
+=====================
+Regressing out the standard six motion parameters may not be sufficient to remove all variance related to head motion
```

Comment:
I associate the term "Regressing out" more with denoising. But this applies to both GLM and denoising right? I've tried to make it more neutral: 
```suggestion
Only accounting for the standard six motion parameters may not be sufficient to remove all variance related to head motion
```